compiler/rustc_lint/src/nonstandard_style.rs

```diff
@@ -138,6 +138,8 @@ impl NonCamelCaseTypes {
                         to_camel_case(name),
                         Applicability::MaybeIncorrect,
                     );
+                } else {
+                    err.span_label(ident.span, "should have an UpperCamelCase name");
```

Comment:
Oh, these labels are nice :)

Reply:
Should this use the ["proper" name](https://medium.com/better-programming/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841) instead, or would that be more confusing?
```suggestion
                    err.span_label(ident.span, "should have an PascalCase name");
```

Reply:
We use "upper camel case" in this context so it should be out of scope anyway.